squidDrone/tests/mock_libraries/com/com_message_buffer_mock.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef TESTS_MOCK_LIBRARIES_COM_COM_MESSAGE_BUFFER_HPP_
+#define TESTS_MOCK_LIBRARIES_COM_COM_MESSAGE_BUFFER_HPP_
+
+#include <queue>
+#include <array>
+
+namespace com 
+{
+
+  class ComMessageBuffer {
+    public:
+      ComMessageBuffer(){
+        _data.fill(0xaa);
+        test_member = 0;
+      };
+      std::uint8_t put_data(std::array<std::uint8_t, 32> &data);
+      std::array<std::uint8_t, 32> get_data();
+
+      std::uint8_t test_member;
+
+    protected:
+      std::array<std::uint8_t,32> _data;
```

Comment:
underscore at the end of the variable name for indicating that it is a private class member